twisted/test/test_iutils.py

```diff
@@ -121,8 +120,8 @@ def test_outputAndValue(self):
 
         def gotOutputAndValue(out_err_code):
             out, err, code = out_err_code
-            self.assertEqual(out, "hello world!\n")
-            self.assertEqual(err, "goodbye world!" + os.linesep)
+            self.assertEqual(out, b"hello world!\n")
+            self.assertEqual(err, b"goodbye world!" + os.linesep.encode())
```

Comment:
Can we use `.encode('ascii')` please? I don't want us to trip up over weird encodings.